packages/component-library-react/src/Blockquote.test.tsx

```diff
@@ -0,0 +1,108 @@
+import { render } from '@testing-library/react';
+import { createRef } from 'react';
+import { Blockquote } from './Blockquote';
+import '@testing-library/jest-dom';
+
+describe('Blockquote', () => {
+  it('renders an HTML blockquote element', () => {
+    const { container } = render(<Blockquote />);
+
+    const blockquote = container.querySelector('blockquote:only-child');
+
+    expect(blockquote).toBeInTheDocument();
+  });
+
+  it('renders a block element', () => {
+    const { container } = render(<Blockquote />);
+
+    const paragraph = container.querySelector(':only-child');
+
+    expect(paragraph).toHaveStyle({ display: 'block' });
+  });
+
+  it('renders a design system BEM class name: utrecht-blockquote', () => {
+    const { container } = render(<Blockquote />);
+
+    const paragraph = container.querySelector(':only-child');
+
+    expect(paragraph).toHaveClass('utrecht-blockquote');
+  });
+
+  it('renders rich text content', () => {
```

Comment:
I would propose checking if it allows for multiple paragraphs as rich text as I think just one might give a false positive in some implementations where the blockquote always contains a paragraph anyway 🤓